packages/procaptcha-frictionless/src/ProcaptchaFrictionless.tsx

```diff
@@ -12,21 +12,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import { isBot } from "@prosopo/detector";
+import { ProviderApi } from "@prosopo/api";
+import detect from "@prosopo/detector";
+import { getRandomActiveProvider } from "@prosopo/procaptcha-common";
 import { ProcaptchaPow } from "@prosopo/procaptcha-pow";
 import { Procaptcha } from "@prosopo/procaptcha-react";
-import type {
-	BotDetectionFunction,
-	ProcaptchaFrictionlessProps,
+import {
+	type BotDetectionFunction,
+	type FrictionlessState,
+	type ProcaptchaClientConfigOutput,
+	ProcaptchaConfigSchema,
+	type ProcaptchaFrictionlessProps,
 } from "@prosopo/types";
 import { ProcaptchaPlaceholder } from "@prosopo/web-components";
 import { useEffect, useState } from "react";
 
-const customDetectBot: BotDetectionFunction = async () => {
-	return await isBot().then((result) => {
-		const bot = result.isBot;
-		return { bot };
-	});
+const customDetectBot: BotDetectionFunction = async (
+	config: ProcaptchaClientConfigOutput,
+) => {
+	const botScore: { token: string } = await detect();
+
+	if (!config.account.address) {
+		throw new Error("Hugh resolve this error!");
+	}
+
+	// Get random active provider
+	const provider = await getRandomActiveProvider(config);
+	const providerApi = new ProviderApi(
+		provider.provider.url,
+		config.account.address,
+	);
+
+	if (!config.account.address) {
+		throw new Error("Hugh resolve this error!");
```

Comment:
We can't merge this - why would it happen?

Reply:
Resolved